### PR TITLE
site: add v1.12.0 blog post

### PR DESCRIPTION
**Pull request by skriss**
Signed-off-by: Steve Kriss <krisss@vmware.com>

cc @jonasrosland

### CODE REVIEW

**Review by youngnick — APPROVED** [review]
LGTM, should wait for @jonasrosland to review though.

File: site/_posts/2021-01-29-contour_v1120.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Contour 1.12 also adds support for including dynamic values in configured request and response headers.
+Almost all [variables supported by Envoy](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#custom-request-response-headers) are allowed.
+This feature can be used to set headers containing information such as the host name of where the Envoy pod is running, the TLS version, etc.
+
+For more information, including a full list of supported variables, see the [Contour documentation](https://projectcontour.io/docs/v1.12.0/config/request-rewriting/#dynamic-header-values).
+
+A big thanks to [@erwbgy](https://github.com/erwbgy) for designing and implementing this feature!
+
+## TODO Header Hash Load Balancing
+<will add if this makes it into the release>
```
Comment (jonasrosland) [inline site/_posts/2021-01-29-contour_v1120.md:70]: Just make sure to remove or add content here :)

**Review by jonasrosland — APPROVED** [review]
LGTM, just one comment, thanks Steve!

**Review by stevesloka — APPROVED** [review]

File: site/_posts/2021-01-29-contour_v1120.md (hunk 1 above)
Comment (skriss) [reply site/_posts/2021-01-29-contour_v1120.md]: done

**Review by sunjayBhatia — APPROVED** [review]

**Pull request merged**